### PR TITLE
add ideal to v2v3 transform

### PR DESCRIPTION
**Pull request by nden**
Implements transforms between the telescope Ideal and V2,V3 coordinate systems.
Also adds XOFFSET, YOFFSET to the core schema.

### CODE REVIEW

File: jwst/transforms/models.py [hunk 1]
```diff
@@ -813,6 +813,122 @@ def __call__(self, v2, v3):
...
+            Parity.
+
+        Returns
+        -------
+        v2, v3 : ndarray-like
+            Coordinates in the (V2, V3) telescope system [in deg].
+
+        """
+        v2ref = np.deg2rad(v2ref / 3600)
+        v3ref = np.deg2rad(v3ref / 3600)
```
Comment (hbushouse) [inline jwst/transforms/models.py]: Up above in Parameters you indicate that v2ref and v3ref are in degrees, yet here it looks like you're assuming the input values will be in arcsec and are being converted to degrees.

File: jwst/transforms/models.py [hunk 2]
```diff
@@ -813,6 +813,122 @@ def __call__(self, v2, v3):
...
+        v3idlyangle = np.deg2rad(v3idlyangle)
+        xidl = np.deg2rad(xidl)
+        yidl = np.deg2rad(yidl)
```
Comment (hbushouse) [inline jwst/transforms/models.py]: For the case that we need this transform (converting the XIdl, YIdl values from PPS) the inputs will be in units of arcsec. Here you're assuming they're in degrees. Will the inputs be converted from arcsec to degrees before being sent to this function?

File: jwst/transforms/models.py [hunk 3]
```diff
@@ -813,6 +813,122 @@ def __call__(self, v2, v3):
...
+        vparity : int
+            Parity.
+
+        Returns
+        -------
+        xidl, yidl : ndarray-like
+            Coordinates in the Ideal telescope system [in deg]. 
+
+        """
+        v2ref = np.deg2rad(v2ref / 3600)
```
Comment (hbushouse) [inline jwst/transforms/models.py]: Same comment regarding degrees vs. arcsec

**Comment by nden** [comment]
Thanks @hbushouse. Units fixed and tests passing now - had to update a couple test files.

**Comment by hbushouse** [comment]
:1st_place_medal:

**Pull request merged**